content/documentation/index.md

```diff
@@ -38,15 +38,15 @@ The core principles underlying NATS are performance, scalability, and ease-of-us
 
 NATS is a simple yet powerful messaging system designed to natively support modern cloud architectures. Because complexity does not scale, NATS is designed to be easy to use and implement, while offering multiple qualities of service.
 
-Some of the types of use cases that are ideal for NATS include:
-
-- High througput message fanout
-- Addressing, discovery
-- Command and control (control plane)
-- Load balancing
-- N-way scalability
-- Location transparency
-- Fault tolerance
+Some of the use cases and requirements that are ideal for NATS include:
+
+- **High throughput message fanout** - a small number of data producers (publishers) need to frequently send data to a much larger group of consumers (subscribers), many of whom share common interest in specific data sets or categories (subjects)
+- **Addressing, discovery** - sending data to specific application instances, devices, or users, or discovering all application instances/devices/users that are connected to your infrastructure. 
+- **Command and control (control plane)** - sending commands to running applications/devices and receiving status back from applications/devices, e.g. SCADA, satellite telemetry, IOT.
+- **Load balancing** - your application(s) produces a large volume of work items or requests and you would like to use a dynamically scalable pool of worker application instances to ensure you're meeting SLAs or other performance targets.
```

Comment:
Are you suggesting that as a new bullet item, or elaboration on 'load balancing' ?